GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -211,16 +213,27 @@ protected override void PerformMaintenance()
                         }
                     }
 
-                    int beforeLooseObjectsCount = this.CountLooseObjects();
+                    this.CountLooseObjects(out int beforeLooseObjectsCount, out long beforeLooseObjectsSize);
+                    this.GetPackFilesInfo(out int beforePackCount, out long beforePackSize, out bool _);
                     GitProcess.Result gitResult = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot), nameof(GitProcess.PrunePacked));

```

Comment:
I don't think this `gitResult` is being used anywhere - is there any decisions we should make on the result (success or failure) of this operation or record the result to telemetry?